modules/auxiliary/scanner/sap/sap_web_gui_brute_login.rb

```diff
@@ -183,6 +198,39 @@ def bruteforce(uri,user,pass,cli)
       elsif res.body =~ /Password logon no longer possible - too many failed attempts/
         print_error("[SAP] #{rhost}:#{rport} - #{user} locked in client #{cli}")
         return false
+      elsif res.body =~ /Logon&#x20;cookie&#x20;check&#x20;failed&#x3b;&#x20;repeat&#x20;logon/ or res.body =~ /logon&#x20;cookie&#x20;is&#x20;missing/
+        xsrf_cookie = res.headers['set-cookie'].match(/sap-login-XSRF_([a-zA-Z0-9]+)=(.*);/)
```

Comment:
From `msftidy`:
```
[WARNING] Do not read Set-Cookie header directly, use res.get_cookies instead: xsrf_cookie = res.headers['set-cookie'].match(/sap-login-XSRF_([a-zA-Z0-9]+)=(.*);/)
```

Reply:
Whoops, just saw your original note above. This is probably a bug in `Rex` that needs to be fixed, since HTTP is case-insensitive.